contracts/stake/src/stake.rs

```diff
@@ -0,0 +1,91 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use crate::*;
+
+use canonical_derive::Canon;
+use dusk_bytes::Serializable;
+
+#[derive(Debug, Default, Clone, Copy, Canon, PartialEq, Eq)]
+pub struct Stake {
+    value: u64,
+    eligibility: u64,
+    expiration: u64,
+}
+
+impl Stake {
+    pub const fn new(value: u64, eligibility: u64, expiration: u64) -> Self {
```

Comment:
It is fine and makes sense to keep the `expiration` parameter, but keep in mind that the latest requirements for the Stake got rid of the `expiration` time